table/options.go

```diff
@@ -336,6 +336,16 @@ func (m Model) WithFilterInputValue(value string) Model {
 	return m
 }
 
+// WithFilterFunc adds a filter function to the model. If the function returns a
+// truthy value, the row will be included in the filtered results. If the function
+// is nil, all rows will be included. The filter input is passed as the second
```

Comment:
Will all rows be included, though?  I thought the behavior was if the function is nil, the filter func wouldn't be used and the current filtering behavior would happen instead.

Reply:
Updated the doc for more clarity.